inclusive-design/idrc
# feat: add default open graph image (resolves #9)

## Description

Adds a default Open Graph image.

## Steps to test

1. Test deploy preview in an Open Graph sharing debugger.

**Expected behavior:** Default image is shown if no custom social image is set.

## Additional information

Not applicable.

## Related issues

- Resolves #9.
